protocol/wakusync/progress_response.go

```diff
@@ -9,6 +9,11 @@ type FetchingBackupedDataDetails struct {
 	TotalNumber uint32 `json:"totalNumber,omitempty"`
 }
 
+type FetchingBackedUpDataTracking struct {
```

Comment:
It is not used in any kind of response. Not sure if it is a proper place for it (could be messenger's implementation detail).

Reply:
Not sure. I put it with the other Backup type. I'm open to move it though

Reply:
I agree, perhaps just put it in the `messenger_backup.go` or smth. Probably together with the `BackupFetchStatus` that I proposed